ajax/libs/snabbdom/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "snabbdom",
+  "version": "0.5.4",
+  "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/paldepind/snabbdom.git"
```

Comment:
@kennynaoh Ok, I've change the url to `git+https://github.com/snabbdom/snabbdom.git`